src/main/java/frc/robot/pose/RobotPoseManager.java

```diff
@@ -37,6 +39,8 @@ public void updateEncoders(EncoderValues newEncoderValues) {
     }
 
     public void updateVisionData(VisionData newVisionData) {
+        count += 1;
+        SmartDashboard.putNumber("Number of visionUpdates", count);
```

Comment:
did we ever prove that logging is fine? ie, can we switch this back to logger.log()